### PR TITLE
Upgraded to django 1.8

### PR DESCRIPTION
**Pull request by manjunath24**

### CODE REVIEW

**Comment by manjunath24** [comment]
Hi @shabda : Can you review this once?

File: blogango/forms.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ class PreferencesForm(forms.ModelForm):
 
     class Meta:
         model = Blog
+        exclude = ()
```
Comment (shabda) [inline blogango/forms.py]: why is this necessary?

Comment (manjunath24) [inline blogango/forms.py]: From django 1.8, you must provide either fields or exclude in model forms.

Comment (shivakrshn49) [inline blogango/forms.py]: I think we need to use fields attribute over there instead of exclude = () like fields = "__all__" as from django 1.8 declaring a model inside meta class also needs to specify an extra attribute fields or exclude

File: blogango/forms.py [hunk 2]
```diff
@@ -50,6 +50,7 @@ class PreferencesForm(forms.ModelForm):
 
     class Meta:
         model = Blog
+        fields = '__all__'
```
Comment (shabda) [inline blogango/forms.py]: @manjunath24 : "Explicit is better than implicit". Lets list the fields needed explicitly.

File: blogango/forms.py (hunk 1 above)
Comment (manjunath24) [inline blogango/forms.py]: @shabda : :thumbsup:
